tests/test_audiofile.py

```diff
@@ -190,7 +190,7 @@ def test_read(tmpdir, duration, offset):
 )
 @pytest.mark.parametrize(
     'empty_file',
-    ('bin', 'mp3', 'wav'),
+    ('bin', 'mp4', 'wav'),
```

Comment:
Why do we use mp4 instead of mp3 here?

Reply:
The input to the test was designed to represent files for three categories of files that are handled differently by `audiofile`:
* non-audio files (`'bin'`)
* audio files not supported by `soundfile` (before `'mp3'`, now `'mp4'`)
* audio files supported by `soundfile` (`'wav'`)

Hence, I switched from `'mp3'` to `'mp4'` in a few tests.

Reply:
I see, thanks for the detailed explanation!